tuf/api/metadata.py

```diff
@@ -708,6 +720,14 @@ def __init__(
         hashes: Optional[Dict[str, str]] = None,
         unrecognized_fields: Optional[Mapping[str, Any]] = None,
     ) -> None:
+
+        if version <= 0:
```

Comment:
Nit: these days I had a thought that it's probably easier to read if we use `if version < 1` when doing this check.
What do you think?

Reply:
I find `<= 0` very readable